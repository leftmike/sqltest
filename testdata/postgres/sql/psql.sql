--
-- Test psql output format
--

DROP TABLE IF EXISTS tbl1;

CREATE TABLE tbl1 (c1 text primary key, integer_column int, c3 text);

INSERT INTO tbl1 VALUES
    ('one', 1, NULL),
    ('two', 2, 'two two two two two two two'),
    ('two hundred', 200, 'two hundred');

SELECT * FROM tbl1;
