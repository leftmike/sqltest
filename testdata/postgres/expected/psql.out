DROP TABLE
CREATE TABLE
INSERT 0 3
c1|integer_column|c3
one|1|
two|2|two two two two two two two
two hundred|200|two hundred
(3 rows)
